engine/data/transforms/_transforms.py

```diff
@@ -70,6 +70,12 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         fill = self._fill[type(inpt)]
         padding = params['padding']
         return F.pad(inpt, padding=padding, fill=fill, padding_mode=self.padding_mode)  # type: ignore[arg-type]
+    
+    # Compatibility with torchvision>0.20.0
+    def transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
```

Comment:
Instead of duplicating the code I would just call self._transform()

```python
 def transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
        return self._transform(inpt, params)